.travis.yml

```diff
@@ -117,3 +117,73 @@ jobs:
         - make edgeimage ARCH="arm64v8"
         - make edgesiteimage ARCH="arm64v8"
         - make bluetoothdevice_image
+
+  - stage: Test on amd64 and build Debian Packages for amd64, aarch64 and armv7l
+      name: build, smallbuild, crossbuild
+      arch: amd64
+      before_script:
+      - sudo apt-get install upx-ucl -y
```

Comment:
In addition to the comment above, apt-get would not work on OSX, you may consider running `fpm` inside a container

Reply:
You don't have to execute this on your local system. This build stage will build the .deb files and add those into the release package; No manual step is required.

Reply:
why is upx-ucl required on building the binary packages?